pxtrunner/runner.ts

```diff
@@ -466,6 +466,14 @@ namespace pxt.runner {
         simDriver?.postMessage(msg);
     }
 
+    export async function fetchSimJsInfo(simOptions: SimulateOptions): Promise<pxtc.BuiltSimJsInfo> {
+        try {
+            return await pxt.Cloud.downloadBuiltSimJsInfoAsync(simOptions.id);
+        } catch {
+            return undefined;
```

Comment:
maybe add a console warn here

Reply:
Sure. Since most games won't be precompiled, the exception path will execute here virtually every time this is called, but logging it could be useful for debugging.